Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Driver.hs

```diff
@@ -410,3 +416,8 @@ clearOnRideStuckDrivers :: ShortId DM.Merchant -> ApiTokenInfo -> FlowHandler Co
 clearOnRideStuckDrivers merchantShortId apiTokenInfo = withFlowHandlerAPI $ do
   checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
   Client.callDriverOfferBPP checkedMerchantId (.drivers.clearOnRideStuckDrivers)
+
+bookingWithVehicleNumberAndPhone :: ShortId DM.Merchant -> ApiTokenInfo -> Common.BookingWithVehicleAndPhoneReq -> FlowHandler APISuccess
```

Comment:
It should be part of Ride APIs. Also, add buildTransaction in this.

Reply:
ok